packages/token-tools/src/color.ts

```diff
@@ -183,3 +190,130 @@ export function tokenToCulori(value: ColorValueNormalized): Color | undefined {
     }
   }
 }
+
+export interface ColorSpaceDefinition {
+  ranges: [min: number, max: number][];
+}
+
+/** Complete list of CSS Module 4 Colorspaces */
+export const COLORSPACE: Record<ColorSpace, ColorSpaceDefinition> = {
+  'a98-rgb': {
+    ranges: [
+      [0.0, 1.0],
+      [0.0, 1.0],
+      [0.0, 1.0],
+    ],
+  },
+  'display-p3': {
+    ranges: [
+      [0.0, 1.0],
+      [0.0, 1.0],
+      [0.0, 1.0],
+    ],
+  },
+  hsl: {
+    ranges: [
+      [0.0, 360.0],
+      [0.0, 100.0],
```

Comment:
The ranges here may be a little overkill. But I wanted to avoid having this hardcoded somewhere that’d be easy to miss elsewhere.